examples/paho-mqtt/README.md

```diff
@@ -0,0 +1,40 @@
+### About
+This application demonstrates the usage of the MQTT paho package in RIOT.
+
+### Setup
+For using this example, two prerequisites have to be fulfilled:
+
+1. You need a running MQTT broker like Mosquitto broker for example. Take a look at
+[Mosquitto Broker](https://mosquitto.org/). Check online any guide that will help you setting up the broker into some port (a).
+For example this one for debian base linux users [How to setup a Mosquitto MQTT Server and receive data](https://www.digitalocean.com/community/questions/how-to-setup-a-mosquitto-mqtt-server-and-receive-data-from-owntracks).
+
+2. Your RIOT node needs to be able to speak to that broker at the same port you set in 1.
+
+### Setting up RIOT `native` on Linux
+- Run the script create.sh in dist/tools
+- Replace the value of *DEFAULT_MQTT_BROKER_IP* with the IPv6 address provide by `ifconfig tapbr0`
```

Comment:
I would drop this line and explain the `ifconfig tapbr0` in the `Running the example` section.
You can also drop the lines about port and the one after.

But I would just give an instruction for running the native instance:
```
make -C examples/paho-mqtt all term
```